src/write.rs

```diff
@@ -330,6 +357,52 @@ where
     pub fn raw_write_handle(&mut self) -> NonNull<T> {
         self.w_handle
     }
+
+    /// Returns the backing data structure.
+    ///
+    /// Makes sure that all the pending operations are applied and waits till all the read handles
+    /// have departed. Then it uses `Absorb::drop_first` to drop one of the copies of the data and
+    /// returns the other copy as an owned value.
+    ///
+    /// ## With custom `drop_second`:
+    ///
+    /// This will not call `Absorb::drop_second` for you!
+    ///
+    /// If you implemented a custom `Absorb::drop_second` then, instead of letting your `T` drop, you should:
+    /// ```
+    /// use left_right::Absorb;
+    ///
+    /// #[derive(Debug, Clone, PartialEq)]
+    /// struct BackingData(i32);
+    /// # struct CounterAddOp(i32);
+    /// # impl left_right::Absorb<CounterAddOp> for BackingData {
+    /// #    fn absorb_first(&mut self, operation: &mut CounterAddOp, _: &Self) {
+    /// #        self.0 += operation.0;
+    /// #    }
+    /// #
+    /// #    fn sync_with(&mut self, first: &Self) {
+    /// #        self.0 = first.0
+    /// #    }
+    /// # }
+    ///
+    /// let original_data = BackingData(10);
+    /// let (write, read) = left_right::new_from_empty::<_, CounterAddOp>(original_data.clone());
+    ///
+    /// let taken_data = write.take();
+    /// assert_eq!(original_data, taken_data);
+    ///
+    /// // make sure it is dropped using `drop_second` if your type requires it.
+    /// Absorb::drop_second(Box::new(taken_data));
```

Comment:
I think this should be `BackingData::drop_second(` just for clarity.

Reply:
I thought `Absorb::drop_second` was clearer but I'll change it.

Reply:
I removed that whole part about the safety because it is handled by the smart pointer.